### DISCUSSION TITLE
Can you recommend a card that can be sent and receive at 1000mw power?

### DISCUSSION THREAD

**Discussion by jiratto** [svpcom/wfb-ng#239] [Questions] [answered]
Can you recommend a card that can be sent and receive at 1000mw power?

Thank you

**Comment by svpcom** [comment, marked as answer]
Officially wifi cards unable to do TX with 30dBm due to CRDA limits. But some cards with **proper cooling** can do 500mW. For example "taobao card" and alfa awus036ach. https://openhd.gitbook.io/open-hd/hardware/wifi-adapters

**Reply by jiratto** [reply]
Thank you
